app/src/main/java/com/example/whattowatch/fragments/RecommenderFragment.java

```diff
@@ -5,16 +5,49 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
 
+import com.codepath.asynchttpclient.AsyncHttpClient;
+import com.codepath.asynchttpclient.callback.JsonHttpResponseHandler;
 import com.example.whattowatch.R;
+import com.example.whattowatch.adapters.MovieListAdapter;
+import com.example.whattowatch.adapters.RecommenderAdapter;
+import com.example.whattowatch.models.Movie;
+import com.example.whattowatch.models.MovieList;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import org.json.JSONException;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import okhttp3.Headers;
 
 
 public class RecommenderFragment extends Fragment {
+    public static final String TAG = "MovieListFragment";
```

Comment:
what is this tag used for?

Reply:
That tag should be "RecommenderFragment". That code is outdated, as I changed it yesterday once I caught that. TAGs are used for logging. Whenever we call Log, which puts code into our logcat, we tag everything we input, so typically it is common to have a TAG variable for each java file so you can identify what file each line in the log cat comes from